libraries/botbuilder-dialogs-adaptive-testing/tests/testTelemetryProperties.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Return expected properties for codeIntent utterance 'intent a1 b2'.
+ *
+ * @returns {*} Object of expected telemtry properties.
+ */
+const getCodeIntentProperties = () => {
+    return {
+        TopIntent: 'codeIntent',
+        TopIntentScore: '1',
+        Intents: '{"codeIntent":{"score":1,"pattern":"(?<code>[a-z][0-9])"}}',
```

Comment:
I think these would be more readable if you used `JSON.stringify` inline and actually wrote these as objects. Simpler to visually parse and update, if necessary, in the future.

```typescript
return {
  foo: JSON.stringify({
    bar: {
      ...
    }
  }),
}
```

Reply:
Great idea! *Much* more readable. Updated to use JSON.stringify